cli/ceph/ceph_volume/lvm.py

```diff
@@ -0,0 +1,59 @@
+from cli import Cli
+
+
+class Lvm(Cli):
+    """This module provides CLI interface to manage the ceph-volume plugin."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Lvm, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} lvm"
+
+    def prepare(self, data, bluestore=True, dmcrypt=False, **kw):
+        """
+        Adds metadata to logical volumes
+        Args:
+            data(str) : Block/raw device
+            bluestore(boolean) : BlueStore is the default backend.
+            Ceph permits changing the backend, which can be done by using the --bluestore flag
+            dmcrypt(boolean) : For enabling encryption, the --dmcrypt flag is required
+        """
+        cmd = f"{self.base_cmd} prepare --data {data}"
+        if bluestore:
+            cmd += " --bluestore"
+        if dmcrypt:
+            cmd += " --dmcrypt"
+        out = self.execute(cmd=cmd, sudo=True, **kw)
```

Comment:
this will only allow single node, for multinode refer - https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/osd/crush.py#L43

Reply:
Done